frontend/components/auction/surplus/SurplusAuctionTransactionFlow.stories.js

```diff
@@ -57,43 +62,40 @@ const common = {
             }, 1000);
         },
         bid(amount) {
-            this.isBidding = true;
+            this.auctionActionState = 'bidding';
             setTimeout(() => {
-                this.isBidding = false;
                 this.auction = {
                     ...this.auction,
                     receiverAddress: this.walletAddress,
                     bidAmountMKR: amount,
                 };
                 action('bid', amount);
+                this.auctionActionState = undefined;
             }, 1000);
         },
         collect() {
-            this.isCollecting = true;
+            this.auctionActionState = 'collecting';
             setTimeout(() => {
-                this.isCollecting = false;
+                this.auction.state = 'collected';
+                this.auctionActionState = undefined;
             }, 1000);
         },
     },
     template: `
         <SurplusAuctionTransactionFlow 
           v-bind="$data" 
-          @connect="connect" 
-          @disconnect="disconnect" 
+          @connectWallet="connect" 
+          @disconnectWallet="disconnect" 
           @setAllowanceAmount="setAllowanceAmount" 
-          @refresh="refresh" 
-          @bid="bid"
+          @refreshWallet="refresh" 
+          @bid="bid(amount)"
```

Comment:
This will use `this.amount` as a value, while what you actually need to pass is 
```suggestion
          @bid="bid($event)"
```
or (which is equivalent):
```suggestion
          @bid="bid"
```